src/modules/claims/claims.service.ts

```diff
@@ -1354,22 +1360,37 @@ export class ClaimsService {
     );
   }
 
-  /**
+    /**
+   * Verifies that credential was issued by authorized issuer
+   *
+   * @param {VerifiableCredential<RoleCredentialSubject} vc to be verified
+   */
+     async verifyVc(vc: VerifiableCredential<RoleCredentialSubject>) {
```

Comment:
I think it would be good to:
- Add the return type
- Add to the documentation that the expected behaviour is that errors are thrown if verification passes (it would be good to include this in the tests as well)